atoma-service/src/handlers/mod.rs

```diff
@@ -557,7 +616,34 @@ mod vllm_metrics {
         let mut futures: FuturesUnordered<_> = chat_completions_service_urls
             .iter()
             .map(|(chat_completions_service_url, job_name)| {
-                get_metrics(&HTTP_CLIENT, job_name, chat_completions_service_url)
+                let vllm_future = get_vllm_request_queue_time_seconds(
```

Comment:
Can't we do a match instead of querying two services ? It is unlikely that we would have sglang and vllm simultaneously running, even though there is a slim chance